docs/hub/_sections.yml

```diff
@@ -19,8 +19,17 @@
 - local: spaces
   title: Spaces documentation (beta)
 
-- local: adding-a-library
-  title: Integrating a library to the Hub
+- local: tutorial-add-library
```

Comment:
I think we should replace `adding-a-library` entirely but keeping the same name.

Reply:
No strong opinion so if you prefer to have the tutorial prefix it's fine